index.rst

```diff
@@ -491,6 +491,25 @@ take :math:`A` or :math:`B` two cycles.
 BCWP and BCWS are both :math:`$6X`. At this point, the project is complete:
 there is no schedule variance, and a cost variance of :math:`+$1.5X`.
 
+Epic-Based Long Term Plans
+--------------------------
+
+As per :ref:`sec-cycle-plan`, the :term:`epic` is the standard level of
+granularity for planning work over the relatively short term (periods of
+several months). However, epics may also be valuable for longer-term,
+fine-grained planning. When a detailed description of work for a given
+planning package is known, it can and should be described in JIRA through a
+series of epics assigned to the appropriate cycles. These epics can be freely
+created and changed at any time, without any sort of approval process. Of
```

Comment:
I'd make it clear here that these epics can be freely changed without approval "as long as they are not part of an active cycle"